data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -533,7 +533,7 @@
     "type": "furniture",
     "id": "f_NMR",
     "name": "nuclear magnetic resonance spectrometer",
-    "description": "This is a giant electromagnet in a kind of sci-fi looking housing.  Somehow it can be used to wiggle molecular bonds or something, and from there, look at the deepest inner workings of chemical structures!  Magnets: how do they work?",
+    "description": "This is a giant electromagnet with carefully tuned measurement equipment used to observe how magnetic fields interact with nuclear particles.",
```

Comment:
I would change to :
"This is a giant electromagnet used to observe how  nuclear spins are affected by a magnetic field. The workhorse for the characterization of organic molecules." 
Alternatively: "[...] workhorse for the discovery of chemical structures"